packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -44,6 +45,8 @@ export interface SliderProps extends Omit<React.HTMLProps<HTMLDivElement>, 'onCh
   inputAriaLabel?: string;
   /* Aria label for the thumb */
   thumbAriaLabel?: string;
+  /* Adds a tooltip over the thumb containing the current value */
+  thumbHasTooltip?: boolean;
```

Comment:
Can we just call this `hasTooltip`.  This is more consistent with our naming practices.  There is also no other tooltip so the name will not conflict.

Reply:
@KKoukiou @tlabaj 
I think referencing _what_ has the tooltip in the prop name is wise - maybe `hasTooltipOverThumb` is descriptive and follows the naming patterns?

Reply:
Fixed! Sorry for the wrong time to reply here.